### DISCUSSION TITLE
sftpgo on docker seems slow and takes a little to load the contents

### DISCUSSION THREAD

**Discussion by Destrocamil** [drakkan/sftpgo#886] [Q&A]
At first, I used the binary on a EC2 instance. It was very fast, very responsive and the downloads speed were very nice.
Now, I moved into docker and it seems to be much slower, taking 1, 2 seconds to load the contents and the download speeds seems to be capped at 1,2 MB/s.
The machine is the same, the difference is that is used docker instead of the binary.
I know that using docker is not the same as using the binary, however I would just like to know if there is anything I can do about, some settings for sftpgo or similar.
Thanks!

**Comment by drakkan** [comment]
There is no docker specific setting. Most SFTPGo users use it on Docker with no issues. This looks like an issue specific to your installation. You have to debug yourself, sorry
